src/DSL/TSBuilder.cs

```diff
@@ -468,6 +468,16 @@ public void Undefined()
             Text("undefined");
         }
 
+        public void Lambda(string paramName, Action<TSBlock> lambdaBodyAction)
+        {
+            Line($"{paramName} => {{");
+            Indent(() =>
+            {
+                lambdaBodyAction.Invoke(new TSBlock(this));
+            });
+            Line($"}}");
```

Comment:
Since a lambda is an expression (as opposed to a statement), the last output should be a `Text()` call, not a `Line()`.